libs/gisday/ngx/src/lib/pages/admin/pages/base-admin-edit/base-admin-edit.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Directive, OnDestroy, OnInit } from '@angular/core';
+
+import { Observable, Subject } from 'rxjs';
+import { shareReplay } from 'rxjs/operators';
+
+import { BaseService } from '@tamu-gisc/gisday/data-access';
+import { GuidIdentity } from '@tamu-gisc/gisday/data-api';
+
+@Directive()
```

Comment:
https://stackoverflow.com/a/47883806
https://angular.io/guide/migration-undecorated-classes

Reply:
https://github.com/angular/angular/issues/40963
I fixed it like the other admin views but this one since it is `<T extends GuidIdentity>` VSCode shows an error but it appears that that is false?